backend/src/types.ts

```diff
@@ -18,6 +18,7 @@ export type DashboardConfig = K8sResourceCommon & {
       disableInfo: boolean;
       disableSupport: boolean;
       disableClusterManager: boolean;
+      disableModelRegistryManager: boolean;
```

Comment:
@andrewballantyne I'll defer this decision to you, there's an admin section for model registry which is an integral part of the feature, do we use the same feature flag for the enablement/disablement of the feature or should we use a different one?

Reply:
@andrewballantyne prefers not adding the extra flag. "sounds like you should have 1 flag. Don’t make it complicated for the user just because"

removed that

Reply:
Great, thanks for adding the reply here!